src/head_jump.h

```diff
@@ -110,6 +114,10 @@ static inline void do_write_cr0(unsigned long val)
 		memcpy((void *)&store_orig_##func, __orig_##func, AARCH64_INSN_SIZE); \
 		store_jump_##func = aarch64_insn_gen_branch_imm((unsigned long)__orig_##func,	\
 				  (unsigned long)prefix##func, AARCH64_INSN_BRANCH_NOLINK); \
+		if ((store_orig_##func & 0xFF) == 0x14 || (store_orig_##func & 0xFF) == 0x94) { \
+			printk("scheduler: Conflicting traced function %s\n", #func); \
```

Comment:
Since we'd like to apply plugsched framework to ebpf, it's better use general term.  
1. use macro KBUILD_MODNAME to replace scheduelr.
2. Conflicting traced function xxx  ----> not clear to me.